### PR TITLE
Subtask - KLBP-44 - Content societe step.

### PR DESCRIPTION
**Pull request by ioan-iacob-netea**
Marked as WIP as a new KYB core version needs to be created and added.

### CODE REVIEW

File: src/lang/fr.json [hunk 1]
```diff
@@ -6,5 +6,14 @@
...
+  "step1.button1.details": "Extrait Kbis, Extrait D1, Extrait RCA ou Situation SIRENE... de moins de 3 mois",
+  "step1.button2.details": "Avis d'imposition (de moins de 1 an) ou information declarative de situation professionnelle",
+"step1.information-box": "<b>Bienvenue sur votre espace de collecte</b> de documents dans le cadre de la remédiation de votre société.{br}Dans ce parcours vous seront demandés tous les documents et donnees necessaires a la <b>verification des information légales constituant votre dossier</b>",
+  "step1.sub-title": "Groupe LGM",
+  "step1.text1": "Entreprise Individuelle",
+  "step1.text2": "Artisan, commerçant, EIRL, Micro-entreprise, Auto-entrepreneur et Indépendant",
+  "step1.text3": "<b>Merci de bien vouloir charger les justificatifs suivants</b>",
+  "step1.title": "Société",
+  "step1.tooltip": "<a>Pourquoi est-ce important?</a>"
+}
```
Comment (mariuspop86) [inline src/lang/fr.json:19]: Add new line

File: src/pages/steps/Step1.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            tag="p"
+            className="mb2"
+            values={{
+              b: (...chunks: any) => <b>{chunks}</b>,
+              br: <br />,
+            }}
+          />
+        </InformationBox>
+        <div>
+          <DynamicFormattedMessage
```
Comment (mariuspop86) [inline src/pages/steps/Step1.tsx:39]:
```suggestion
       <p  className="mb2">
          <DynamicFormattedMessage
            id="step1.text3"
            tag="strong"
          />
        </p>
```
This way you can change 
"step1.text3": "<b>Merci de bien vouloir charger les justificatifs suivants</b>" 
to 
"step1.text3": "Merci de bien vouloir charger les justificatifs suivants",

Try not to use the **<b>** tag it's deprecated

File: src/pages/steps/Step1.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+import { Title, InformationBox } from 'kyb-core'
+import DefaultTemplate from '../templates/DefaultTemplate'
+import DynamicFormattedMessage from '../../components/common/DynamicFormattedMessage'
+
+function Step1(): ReactElement {
+  return (
+    <DefaultTemplate>
+      <div>
+        <Title title="Société">
+          <DynamicFormattedMessage
```
Comment (mariuspop86) [inline src/pages/steps/Step1.tsx:11]:
```suggestion
          <a href="https://www.google.com">{
            <DynamicFormattedMessage id="step1.tooltip" />
          </a>
```
This is way more understandable.
And change 
"step1.tooltip": "`<a>`Pourquoi est-ce important?`</a>`"
to 
"step1.tooltip": "Pourquoi est-ce important?"
in [src/lang/fr.json](src/lang/fr.json)

Avoid to use tags into translations

File: src/pages/steps/Step1.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
+import React, { ReactElement } from 'react'
+import { Title, InformationBox } from 'kyb-core'
+import DefaultTemplate from '../templates/DefaultTemplate'
+import DynamicFormattedMessage from '../../components/common/DynamicFormattedMessage'
+
+function Step1(): ReactElement {
+  return (
+    <DefaultTemplate>
+      <div>
+        <Title title="Société">
```
Comment (andreir-suciu) [inline src/pages/steps/Step1.tsx:10]: `<Title title={
    intl.formatMessage({
      id: 'step1.title',
    }),
    } />`
    
    Please update title translation.

File: src/lang/fr.json (hunk 1 above)
Comment (ioan-iacob-netea) [reply src/lang/fr.json]: Done

File: src/pages/steps/Step1.tsx (hunk 2 above)
Comment (ioan-iacob-netea) [reply src/pages/steps/Step1.tsx]: Done

File: src/pages/steps/Step1.tsx (hunk 3 above)
Comment (ioan-iacob-netea) [reply src/pages/steps/Step1.tsx]: Done

File: src/pages/steps/Step1.tsx (hunk 4 above)
Comment (ioan-iacob-netea) [reply src/pages/steps/Step1.tsx]: Done

File: src/interfaces/Step.ts [hunk 5]
```diff
@@ -1,12 +1,12 @@
-import { FormContextValues } from 'react-hook-form'
+// import { FormContextValues } from 'react-hook-form'
```
Comment (mariuspop86) [inline src/interfaces/Step.ts:1]: Remove this comment

File: src/interfaces/Step.ts [hunk 6]
```diff
@@ -1,12 +1,12 @@
...
 
 export interface IStepProps {
   step: number
   title: string
 }
 
 export interface IUseStepForm {
-  form: FormContextValues
+  form: any
```
Comment (mariuspop86) [inline src/interfaces/Step.ts:9]: Can you try to find a more exact interface?

File: src/interfaces/Step.ts (hunk 5 above)
Comment (ioan-iacob-netea) [reply src/interfaces/Step.ts]: Updated

File: src/interfaces/Step.ts (hunk 6 above)
Comment (ioan-iacob-netea) [reply src/interfaces/Step.ts]: Updated

**Pull request closed without merging**